runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -1,6 +1,7 @@
 package org.corfudb.recovery;
 
 import javax.annotation.Nonnull;
+import java.lang.reflect.Type;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.lang.reflect.Type'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)